os/src/ZipOps.scala

```diff
@@ -0,0 +1,22 @@
+package os
+
+import java.net.URI
+import java.nio.file.{Path => _, _}
+import collection.JavaConverters._
+
+class zip(zipPath: Path) {
+
+  private val fs = FileSystems.newFileSystem(
+    URI.create("jar:file:" + zipPath.wrapped.toString),
+    Map("create" -> "true").asJava
```

Comment:
Could a `Map[String, String]` be an optional parameter to the class constructor with this as the default value? That would make it configurable from the outside.